content/product-engineering/engineering/career-development/framework.md

```diff
@@ -398,63 +401,37 @@ To learn more, see ["Considerations for promotion" in our talent review process]
-        </div>
-      </td>
-    </tr>
-    <tr class="behaviors-row">
-      <td class="behaviors proficiency">
-        <ul>
-          <li>Sets the technical vision for their team, and influences the broader technical vision.</li>
-          <li>Initiates and drives cross-team projects that enable higher quality work.</li>
-          <li>Provides oversight, coaching, and guidance through code and design reviews, both on and off the team.</li>
-          <li>Acts as a trusted advisor, drawing on functional expertise to inform customer-first strategy.</li>
-        </ul>
-      </td>
-      <td class="behaviors execution">
-        <ul>
-          <li>Proactively identifies areas for improvement at the org level. Suggests process and methodology improvements.</li>
-          <li>Works closely with Engineering/Product leadership to validate alignment of team roadmaps within their org.</li>
-          <li>Independently scopes, designs, and delivers solutions for large, complex challenges.</li>
-        </ul>
-      </td>
-      <td class="behaviors teamwork">
+    <tr>
+      <td colspan="3" class="tbd">
+        <p>
+          Rather than being defined through proficiency, execution and teamwork, IC5 is differentiated from levels 1-4 by a specific <strong>business need</strong> for the role, and an associated set of <strong>expectations</strong>.
+        </p>
         <ul>
-          <li>Provides technical expertise internally and externally, informing what can be achieved.</li>
-          <li>Guides others on effective collaboration, conflict resolution and improved communication.</li>
-          <li>Regularly shares knowledge and willingly presents to large and/or senior audiences.</li>
-          <li>Persuades and challenges clients and internal stakeholders, using valid expertise and respectful communication.</li>
+          <li><strong>Business need</strong>: in some cases the business will need an IC to and solve cross-cutting technical problems at the org or company level, willing to navigate across orgs as new problems are identified. In others, it will need a deep technical expert for a specific area, capable of devising solutions to highly complex and specialized technical problems.</li>
+          <li><strong>Expectations</strong>: for some IC5s, expectations might include participating in engineering calibration, because they are expected to define hiring needs for their team. For others, it might be building and owning a roadmap.</li>
         </ul>
+        <p>
+          The business need and associated expectations are unique to each IC5 position. IC5 engineers are publicly identified as such, and the business need and expectations associated with their role are documented in the handbook.
+        </p>
+        <p>
+          Unlike IC4, IC5 is not a terminal level. Transition to the IC5 level is contingent on the existence of a business need for the role.
```

Comment:
When I first read this, it makes it sound like IC5 role might disappear if the need isn't there anymore, not sure if that's the intent